src/executor/load_executor.py

```diff
@@ -42,6 +44,7 @@ def exec(self):
         # We currently use create to empty exsiting table.
         StorageEngine.create(self.node.table_metainfo)
 
-        video_reader = OpenCVReader(self.node.file_path)
+        video_reader = OpenCVReader(
+            os.path.join('/tmp', self.node.file_path))
```

Comment:
`/tmp` is a folder in Ubuntu that hosts temporary files (that are automatically deleted after a certain time). So the name is not a deliberate choice. This design ensures that the videos are in the temporary folder for subsequent consumption by the `LOAD DATA` command. This also ensures that the uploaded videos are temporary on the server, and not taking up real server space. Loading video from any path in the server is not a priority as far as I understand. The `UPLOAD` command which always precedes the `LOAD DATA` command ensures that the video is available on the server. Let me know what you think.

Reply:
I understand how `tmp` works :) I was of the opinion that we are persisting the video on the server. Nevertheless, in case we are going by this design, I think there is no harm in making it a config parameter. We can set it to `/tmp` by default. It will help a lot later. @xzdandy what do you think?

Reply:
Yes. I agree. We can make it a configurable variable in the `eva.yml` file.

Reply:
> I understand how `tmp` works :) I was of the opinion that we are persisting the video on the server. Nevertheless, in case we are going by this design, I think there is no harm in making it a config parameter. We can set it to `/tmp` by default. It will help a lot later. @xzdandy what do you think?

I believe we can't change it to `uploaded` because of how `/tmp` works. Making it a config parameter is trivial :) I will make that change.

Reply:
The latest commit should resolve this. Please review the changes and let me know @gaurav274 @xzdandy

Reply:
Good for me.